build.sbt

```diff
@@ -39,14 +39,13 @@ lazy val commonSettings = {
     ),
     licenses += ("Apache-2.0", new URL("https://www.apache.org/licenses/LICENSE-2.0.txt")),
     organization := "org.apache.daffodil",
-    scalaVersion := "2.12.15",
+    scalaVersion := "2.12.16",
```

Comment:
I think we should not do this update.

Reply:
I will look into seeing about ignoring this via scala-steward